modules/reindex/build.gradle

```diff
@@ -56,6 +56,7 @@ unitTest {
 
 dependencies {
   compile "org.elasticsearch.client:elasticsearch-rest-client:${version}"
+  compile project(':libs:elasticsearch-ssl-config')
```

Comment:
register the ssl-config project in the top-level `build.gradle` under projectSubstitutions, and then you can use `compile "org.elasticsearch:elasticsearch-ssl-config:${version}"`
Also check if the `ssl-config` is properly published through the release manager

Reply:
Thanks @ywelsch, will do.

Reply:
The build.gradle change has been pushed, and there is an open PR for release manager.

Reply:
you're right. No need for that.